common/djangoapps/util/views.py

```diff
@@ -434,6 +440,12 @@ def build_error_response(status_code, field, err_msg):
 
     success = False
     context = get_feedback_form_context(request)
+
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
```

Comment:
my guess around the extra tickets getting tagged in prod is that this isn't always coming back as empty for non enterprise learners, so we might want to check for more than the existence of this return data.